server.js

```diff
@@ -171,7 +175,6 @@ app.post('/comment', function (req, res) {
         }
 
     );
-});
```

Comment:
Missing brace causes server to crash on startup

Reply:
Yeah good catch. The merge removed it.

I added a closing bracket there to fix it.